app/pages/ServiceDiscoveryResults/ServiceDiscoveryResults.tsx

```diff
@@ -205,7 +204,6 @@ const InnerServiceDiscoveryResults = ({
             <div className={styles.results}>
               <SearchResults
                 mobileMapIsCollapsed={isMapCollapsed}
-                categoryId={categoryId}
```

Comment:
Nice, I was going to ask you about that code. My (somewhat strong) preference here would be to not let any ghost code impact any decisions in the present -- unless there's 100% certainty that code will be used/deployed in the immediate term. Case in point: considering today's chat about search it's starting seem somewhat likely the code won't need this property passed here in this way since category pages are not part of the acceptance criteria for search.

I'd suggest we ignore and/or extract this commented code into a Notion doc for safe-keeping so the code can continue to evolve before the presumptive phase 2.

Reply:
It was meant to come back into the picture when the top level category PR was done, which it is now. I'm not totally sure, but I think we should keep it until we have results for those pages.

Other than that I do agree with you, but I think if we remove ghost code, we should remove all related ghost code. There are still references to that categoryId commented out here.

Reply:
@adriencyberspace ask you and you shall receive: [`50365fa` (#96)](https://github.com/Exygy/askdarcel-web/pull/96/commits/50365fad0f3e17adfe1f166e65b0de8ed63634bb). 

Also put the code on ice 🧊 : https://www.notion.so/exygy/Our415-Phase-2-Code-Snippets-bbf33baf3f0c4a38be59c7b9973171e1?pvs=4